builds/assets/lang/frFR.json

```diff
@@ -123,16 +123,16 @@
   "settingsMapsTileServerImageFormat": "Format d'image",
   "settingsMapsMapHeight": "Hauteur de la carte",
   "settingsMapsHybrid": "Hybride",
-  "settingsMapsOverlay": "Recouvrir",
-  "settingsMapsUnderlay": "Thibaude",
-  "settingsMapsBlending": "Mélange",
+  "settingsMapsOverlay": "Au dessus",
+  "settingsMapsUnderlay": "En dessous",
+  "settingsMapsBlending": "Mélangé",
   "settingsMapsOpacity": "Opacité",
   "settingsMapType": "Carte de base",
-  "settingsMapTypeCustom": "Douane",
+  "settingsMapTypeCustom": "Personalisée",
```

Comment:
Should this be different to the Camera Lens Custom just above?

Reply:
It should... weird I did a find and replace ...

Reply:
Sorry Sam I misunderstood your comment and my change was not displayed correctly on my phone.
It is different because FOV in French is masculin but Map is feminin.

Reply:
No worries! I assumed that it was correct, but was kind of curious as to what the difference was. Thanks for explaining it! 😄